tokio-util/tests/stream_reader.rs

```diff
@@ -2,13 +2,14 @@
 #![cfg(feature = "full")]
```

Comment:
Should this file have a `#![warn(rust_2018_idioms)]` on it like the reader stream tests do?

Reply:
It does, the line just isn't visible in the GitHub ui since its too far from changes.